functions/src/scheduler/schedule.ts

```diff
@@ -23,3 +25,29 @@ export const deletePingsAfter30days: any = functions.pubsub.schedule('00 00 * *
     });
     return Promise.all(promises);
   });
+
+export const runAllMonitors60Mins: any = functions.pubsub.schedule('every 60 mins')
+  .onRun(async () => {
+    const snapshots: QuerySnapshot = await FirebaseAdmin.firestore()
+      .collection('projects')
```

Comment:
I think we should filter this out for projects that have `url`s

Reply:
Firestore does not provide `!=` in where clause.
Also there is already a check added for proejct.url before calling `ping` function